src/metabase/util/number.cljc

```diff
@@ -1,19 +1,29 @@
 (ns metabase.util.number
-  "Number parsing helper functions.
-  Most of the implementations are in the split CLJ/CLJS files [[metabase.util.number.impl]]."
+  "Number helper functions that abstract differences between CLJ and CLJS."
   (:refer-clojure :exclude [bigint integer?])
```

Comment:
Would it be possible to add a linter rule to disallow `clojure.core/integer?` and `clojure.core/bigint` outside of this library?

Reply:
It might make sense to do it in `cljc` files. But as we don't normally work with `bigint` this could also create additional noise.